src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -828,16 +843,21 @@ An example SSHPOP provisioner in the `ca.json`:
 - **claims**<Reference id="star5" marker="*" />: overwrites the default `claims` set in the authority,
 see the [claims](#claims) section for all the options.
 
-- **options**<Reference id="star5" marker="*" />: see the [options](#options) section for more details.
+- **options**<Reference id="star5" marker="*" />: see [template configuration](templates#configuring-the-ca-to-use-templates) for more details.
 
 <Footnote id="star5" marker="*">Optional</Footnote>
 
 ### ACME
 
-An ACME provisioner allows a client to request a certificate from the server using the [ACME Protocol](https://tools.ietf.org/html/rfc8555).
-This provisioner supports issuing X.509 certificates with IP or hostname identifiers.
-It supports the `http-01`, `dns-01`, and `tls-alpn-01` ACME challenge types,
-All authentication of the CSR is managed by the ACME protocol.
+The [ACME Protocol](https://tools.ietf.org/html/rfc8555) can authenticate Certificate Signing Requests (CSRs) in a way that enables automation.
+
+ACME clients must answer challenges presented by the ACME server to prove to the CA that they control the identifiers listed in the CSR.
+ACME supports four different types of challenges: `http-01`, `dns-01`, `tls-alpn-01`, and `device-attest-01`. These are designed for operability in different environments.
+See [ACME Basics](/docs/step-ca/acme-basics#acme-challenges) for a description of each challenge type and their tradeoffs.
+
+In a typical setup, an ACME server might issue server certificates via the `http-01`, `dns-01`, `tls-alpn-01` challenge types, and client certificates via the `device-attest-01` challenge type.
```

Comment:
Our default templates allow both uses for the server certs. For sake of brevity, I think it's OK to simplify it as you did